src/backend/utils/compatibility_layers.ts

```diff
@@ -300,33 +300,42 @@ export async function getCrossover(): Promise<Set<WineInstallation>> {
     return crossover
   }
 
+  const crossoverMacPath = new Set<string>()
+
+  // search for crossover installed on /Applications/CrossOver.app
+  const crossoverAppPath = '/Applications/CrossOver.app'
+  if (existsSync(crossoverAppPath)) {
+    crossoverMacPath.add(crossoverAppPath)
+  }
+
+  // search for crossover installed around the system
   await execAsync(
```

Comment:
Have you tested to see if it won't duplicate the entry on the wine list?

Reply:
I cannot test for duplicates as I do not have Spotlight (`mdfind`) enabled. 

However, since `crossoverMacPath` is a `Set`, it should ignore duplicate entries.

Reply:
Yeah, good point. 
I will try to test on a macOS this week just to be sure.